Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: tearup
+tearup:
+	@echo "+ $@"
+	docker-compose up -d
+
+.PHONY: teardown
+teardown:
+	@echo "+ $@"
+	docker-compose rm -fsv
+
+.PHONY: test-run
+test-run:
+	@echo "+ $@"
+	go test -v -p 1 -race -coverprofile=coverage.txt -covermode=atomic ./...
+
+.PHONY: test
+test: tearup test-run teardown
```

Comment:
@Darkemon do we need to add linter here as well, wdyt?

Reply:
I thought about that too, but it'll take take to make the code to conform the linter rules. I suggest to leave it as is to not spend time.